cli/src/semgrep/formatter/json.py

```diff
@@ -30,6 +30,7 @@ def _rule_match_to_CliMatch(rule_match: RuleMatch) -> out.CliMatch:
             lines="".join(rule_match.lines).rstrip(),
             metavars=rule_match.match.extra.metavars,
             dataflow_trace=rule_match.dataflow_trace,
+            is_pro_match=rule_match.match.extra.is_pro_match,
```

Comment:
I suspect we're going to need to differentiate further than this in the future. We'd probably be better off with an enum (I'm expecting we'll want `matcher_engine=SEMGREP|PRO_MULTIFILE|PRO_INTERPROC`)